test/boilerplate/boilerplate.py

```diff
@@ -16,8 +16,8 @@
 # Verifies that all source files contain the necessary copyright boilerplate
 # snippet.
 # This is based on existing work
-# https://github.com/kubernetes/test-infra/blob/master/hack
-# /verify_boilerplate.py
+# https://github.com/kubernetes/test-infra/blob/master/hack/verify_boilerplate.py
```

Comment:
Run the make target this won’t pass the python linter

Reply:
this one gets through just fine now... /shrugh